spec/czCustomizableSpec.js

```diff
@@ -215,11 +221,11 @@ describe('cz-customizable', function() {
       expect(getQuestion(2).when({type: 'WIP'})).toEqual(false);
       expect(getQuestion(2).when({type: 'wip'})).toEqual(false);
 
-      //question 5
-      expect(getQuestion(5).name).toEqual('breaking');
-      expect(getQuestion(5).when({type: 'feat'})).toEqual(true);
-      expect(getQuestion(5).when({type: 'fix'})).toEqual(true);
-      expect(getQuestion(5).when({type: 'FIX'})).toEqual(true);
+      //question 6
+      expect(getQuestion(6).name).toEqual('breaking');
```

Comment:
know what you mean about the test being tide to internal implementation. It is getting a bit messy. It would be good if we didn't have to know indexes. I will think about alternatives.